.travis.yml

```diff
@@ -66,6 +68,7 @@ before_script:
   ## yarp
   - git clone https://github.com/robotology/yarp.git
   - cd yarp
+  - git checkout v3.2.0
```

Comment:
a comment that we are using the latest version and why?

Reply:
https://github.com/robotology/icub-model-generator/pull/119#issuecomment-531726813